pytest_fixtures/component/taxonomy.py

```diff
@@ -141,6 +141,15 @@ def upgrade_entitlement_manifest_org(function_org, upgrade_entitlement_manifest,
     return function_org
 
 
+@pytest.fixture
+def upgrade_sca_manifest_org(function_org, upgrade_sca_manifest, target_sat):
```

Comment:
Could we rename this to `sca_manifest_org_for_upgrade` as the current naming sounds like we are upgrading the sca manifest org.